<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarChartClass</class>
 <widget class="QMainWindow" name="StarChartClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StarChart</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Constelation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="constelationBar"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Right Ascension</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="RABar"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Declination</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="decBar"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Diameter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="diameterBar"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="searchBar"/>
      </item>
      <item>
       <widget class="QTableView" name="starList"/>
      </item>
      <item>
       <widget class="QCheckBox" name="constelationCheck">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Only my constelation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="viewButton">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>View</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="StarChart.qrc"/>
 </resources>
 <connections/>
</ui>
